apps/web/src/pages/CommunityInterests/sections/TrainingAndDevelopmentOpportunities.tsx

```diff
@@ -0,0 +1,285 @@
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1)"
+      >
+        <Heading
+          level="h2"
+          data-h2-font-weight="base(400)"
+          Icon={RectangleGroupIcon}
+          color="primary"
+          data-h2-margin="base(0)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Training and development opportunities",
+            id: "I5K1zI",
+            description:
+              "Heading for the 'Training and development opportunities' section",
+          })}
+        </Heading>
+        <span>
+          {intl.formatMessage({
+            defaultMessage:
+              "Most functional communities offer various programs for learning and development. These programs might have specific eligibility requirements based on your experience, classification, or other qualifications. Expressing interest in these opportunities isn’t an application, but it allows HR and recruitment staff to verify your interest in case you’ve been nominated for a training or development opportunity.",
+            id: "2HNbTn",
+            description:
+              "Description of the 'Training and development opportunities' section",
+          })}
+        </span>
+      </div>
+      {/* list of programs */}
+      <div
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1.5)"
+      >
+        {developmentPrograms.map((developmentProgram, index) => (
+          <div
+            key={developmentProgram.id}
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column)"
+            data-h2-gap="base(x1.25)"
+          >
+            <CardSeparator space="none" />
+            <div
+              key={developmentProgram.id}
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1)"
+            >
+              {/* titles */}
+              <div>
+                <Heading
+                  level="h3"
+                  size="h6"
+                  data-h2-font-weight="base(bold)"
+                  data-h2-margin="base(0)"
+                >
+                  {developmentProgram.name?.localized ??
+                    intl.formatMessage(commonMessages.notProvided)}
+                </Heading>
+                <p>
+                  {developmentProgram.descriptionForProfile?.localized ??
+                    intl.formatMessage(commonMessages.notProvided)}
+                </p>
+              </div>
+              {/* classification list */}
+              {developmentProgram?.eligibleClassifications?.length ? (
+                <div
+                  data-h2-display="base(flex)"
+                  data-h2-flex-direction="base(row)"
+                  data-h2-gap="base(x0.25)"
+                >
+                  <span>
+                    {intl.formatMessage({
+                      defaultMessage: "Available to:",
```

Comment:
Any reason not to use `commonMessages.dividingColon`?

Reply:
It can't hurt.
[dividing colon](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12668/commits/8ead2100e1461d78124d0efb8bdefa116c4b7b0f)